src/redux/types/actions/motion.ts

```diff
@@ -32,6 +32,23 @@ import {
   type UniqueActionTypeWithoutPayload,
 } from './index.ts';
 
+// eslint-disable-next-line no-warning-comments
```

Comment:
we probably don't need this comment anymore since we've removed that addition right?

Reply:
Can I get some clarification about this @bassgeta ? Maybe a quick DM would be easier